wiki/Mapping_Techniques/Rhythm/fr.md

```diff
@@ -0,0 +1,193 @@
+Les objets suivants, pairs, doivent ensuite être placés à 0 degré, 90 degrés, 180 degrés, 270 degrés (ou vice versa) autour du "moyeu" (ce sont les "rayons").
+
+Il est très important de disposer les objets de manière à ce qu'ils aillent dans le sens des aiguilles d'une montre ou dans le sens inverse.
+
+La forme de base du moyeu et rayons utilise 4 rayons.
+Vous pouvez toutefois utiliser 3, 5 ou 6 rayons à la place (dans ce cas, veillez à ce que la mesure du degré reste égale pour chaque rayon).
+
+### Aller-retour linéaires
+
+Nominateur(s) : [Derekku](https://osu.ppy.sh/users/91341)
+
+L'**aller-retour linéaires** est une variation des [battements aller-retour](#battements-aller-retour).
+La différence réside dans le fait que ces derniers n'utilisent pas de piles, mais demandent au joueur de se déplacer entre les objets.
+Il est très important que le mappeur indique clairement la direction dans laquelle se dirige la propagation.
+
+### Spacing Out
+
+*À ne pas confondre avec l'[Espacement accéléré](#espacement-accéléré)*
+
+Nominateur(s) : [Pasonia](https://osu.ppy.sh/users/43345)
+
+Le **spacing out** fait référence aux objets dans lesquels ils sont placés en termes de distance, et non de temps.
+Cette technique a l'avantage de maintenir l'élan d'un morceau pendant cette section de la map.
+Cependant, un patternmapping incorrect peut conduire à une map presque injouable.
+
+En gros, formez un carré autour du centre avec 4 cercles.
+Les 4 cercles suivants doivent ensuite être placés 1 espace de grille plus loin et ainsi de suite.
+
+Une autre variante que vous pouvez ajouter est de placer un cercle au centre après chaque cycle.
+
+### Step-Too-Fars
+
+Nominateur(s) : [OzzyOzrock](https://osu.ppy.sh/users/465153)
+
+Le **step-too-fars** utilise une trajectoire linéaire de cercles qui sont espacés d'un quart de temps.
+Cependant, lorsqu'ils atteignent la "fin" de la ligne, les objets reculent d'une case (le cercle avant la fin de la ligne) puis changent de direction.
+
+## Vitesse
+
+### Ralentissements des sliders
+
+Nominateur(s) : [123MegaApple123](https://osu.ppy.sh/users/2148208)
+
+Les **ralentissements de sliders** ralentissent brusquement un slider en divisant temporairement par deux un point de timing hérité (les verts) pour compléter la note longue de la musique (ou de la voix).
+
+## Streams
+
+*Voir [Techniques de mapping/Basique](/wiki/Mapping_Techniques/Basics#streams) pour une description des streams.*
+
+### Streams compressés
+
+Nominateur(s) : [pieguyn](https://osu.ppy.sh/users/107485)
+
+Les **streams compressés** utilisent des cercles qui ne sont séparés que par 3 espaces de grille (sur un niveau à 2 ou 3 grilles).
+Sinon, il suffit de rendre le stream considérablement plus petit par rapport au reste de la map.
+
+La majorité des maps qui utilisent des streams sont les plus susceptibles d'être vues en utilisant des steams compressés.
```

Comment:
Je suis pas sûr du sens de "To be seen" (peut-être "en fait" dans ce cas là ça donnerait "La majorité des maps qui utilisent des streams sont en fait plus susceptibles d'utiliser des streams compressés.") dans la traduction originale mais je pense qu'on a pas trop besoin de s'en soucier et je pense que ça ne change quasiment rien au sens de la phrase en l'ôtant. Désolé pour le commentaire un peu chargé xD
```suggestion
La majorité des maps qui utilisent des streams sont plus susceptibles d'utiliser des streams compressés.
```